libeos-payg/manager-interface.h

```diff
@@ -131,12 +131,32 @@ static const GDBusPropertyInfo manager_interface_code_format =
   NULL,  /* annotations */
 };
 
+static const GDBusPropertyInfo manager_interface_code_format_prefix =
+{
+  -1,  /* ref count */
+  (gchar *) "CodeFormatPrefix",
+  (gchar *) "s",
+  G_DBUS_PROPERTY_INFO_FLAGS_READABLE,
+  NULL,  /* annotations */
+};
+
+static const GDBusPropertyInfo manager_interface_code_format_sufix =
+{
+  -1,  /* ref count */
+  (gchar *) "CodeFormatSufix",
```

Comment:
```suggestion
  (gchar *) "CodeFormatSuffix",
```
(and everywhere else "sufix" is used)